test/unit/store/map/index/actions.spec.js

```diff
@@ -126,7 +126,24 @@ describe('storeActiveDatasets', () => {
 })
 
 describe('loadPointDataForLocation', () => {
-  test('loads point data for the specified location', async () => {
+  const state = {
+    datasets: {
+      ab: {},
```

Comment:
please use more meaningful properties. This  is a bit vague.

Reply:
Changed it to par1, par2, par3 :)